src/adapter/src/subscribe.rs

```diff
@@ -27,6 +27,59 @@ use timely::progress::Antichain;
 use tokio::sync::mpsc;
```

Comment:
We should rename this file to `compute_sink.rs` and adjust the comment on L10 accordingly.

Reply:
It would show up in the diffs as an entirely new file then. I thought it might be harder to review. Once I get an approval, I will do the rename as the last step before merging?

Reply:
done